riva/client/asr.py

```diff
@@ -117,10 +117,14 @@ def add_audio_file_specs_to_config(
 def add_speaker_diarization_to_config(
     config: Union[rasr.RecognitionConfig],
     diarization_enable: bool,
+    diarization_max_speakers: int=3,
```

Comment:
maybe we can omit 3 here, default value is already set for args.diarization_max_speakers when calling function. In case we update default in future, will have to remember to update at 2 places :)